controllers/servers/utils.py

```diff
@@ -770,16 +770,17 @@ def validate_modify_volume_group_request(request):
 
 
 def get_replication_object_type_and_id_info(request):
+    object_id = request.volume_id
+    object_type = servers_settings.VOLUME_TYPE_NAME
```

Comment:
why is this needed?

Reply:
It's a default case for backward compatibility. In our replication side car a new field for volume group will arrive and handled in switch-case below